contracts/token/ERC721/IERC721.sol

```diff
@@ -1,24 +1,24 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.0;
 
 import "../../introspection/IERC165.sol";
 
 /**
  * @dev Required interface of an ERC721 compliant contract.
  */
-contract IERC721 is IERC165 {
+abstract contract IERC721 is IERC165 {
```

Comment:
We should consider removing the inheritance and making these `interface`. It would also remove the need to make the abstract functions virtual. Doesn't have to be in this PR or in this release.

Reply:
Agreed, added it to the list of pending changes.